### ISSUE TITLE
Periodical crashes

### ISSUE THREAD

**Issue by BEaStia** [codedge-llc/kadabra#32]
Hello!
I'm using 
`
{:pigeon, "~> 1.1.6"},
{:kadabra, "~> 0.3.7"},
`
for sending pushes in phoenix app.
and periodically I see these reports in logs:
```
=SUPERVISOR REPORT==== 27-Mar-2018::09:00:51 ===
     Supervisor: {<0.1849.0>,'Elixir.Kadabra.Supervisor'}
     Context:    child_terminated
     Reason:     normal
     Offender:   [{pid,<0.1853.0>},
                  {id,connection},
                  {mfargs,
                      {'Elixir.Kadabra.Connection',start_link,
                          ["fcm.googleapis.com",<0.1848.0>,<0.1849.0>,
                           #Ref<0.4166094871.989855750.101811>,
                           [{scheme,https},
                            {ssl,
                                [{active,once},
                                 {packet,raw},
                                 {reuseaddr,true},
                                 {alpn_advertised_protocols,[<<"h2">>]},
                                 binary]},
                            {port,443},
                            {reconnect,false}]]}},
                  {restart_type,permanent},
                  {shutdown,5000},
                  {child_type,worker}]
```
What should I do with it ❓

**Comment by hpopp** [comment]
Sorry for the late response. That error message is likely related to the fixes made in #27 and *should* be harmless. 

What Elixir/OTP versions are you running?

**Comment by hpopp** [comment]
Closing this, if it was still an issue, it's been most likely fixed with the `v0.4.3` release.
